.github/workflows/ci.yml

```diff
@@ -183,12 +187,6 @@ jobs:
         uses: golangci/golangci-lint-action@v3.1.0
         with:
           version: latest
-      - name: Format code
```

Comment:
no need for this when we've got golangci-lint doing a gofumpt check. gofumpt is stricter than gofmt